src/extractor/extraction_containers.cpp

```diff
@@ -163,6 +199,7 @@ void ExtractionContainers::PrepareData(ScriptingEnvironment &scripting_environme
     PrepareNodes();
     WriteNodes(file_out);
     PrepareEdges(scripting_environment);
+    NodeVector().swap(all_nodes_list);
```

Comment:
this will not compile  because `stxxl::vector` does not support `shrink_to_fit`. 
I have to check this line actually there is an invalid free issue with stxxl and i guess because of that line.